docs/index.rst

```diff
@@ -14,6 +14,15 @@ Templatekit is available from `PyPI <https://pypi.org/project/templatekit/>`_:
 
    pip install templatekit
 
+Getting started
+===============
+
+.. toctree::
+   :maxdepth: 1
+
+   changelog
```

Comment:
Given there isn't a role for a "Getting started" in the way these docs are structured, the addition of a "Getting started" section here looks really out of place.

I'd suggest rolling it into the installation section until I can think of something better?

```suggestion

`See Templatekit's release history. <changelog>`_.

.. toctree::
   :hidden:
   
   changelog
```